### DISCUSSION TITLE
pf3d* modules moved from icub-main into icub-basic-demos

### DISCUSSION THREAD

**Discussion by pattacini** [robotology/community#50] [Announcements]
Following up https://github.com/robotology/QA/issues/37, `pf3dTracker` and `pf3dBottomup` modules have been moved into [**`icub-basic-demos`**](https://github.com/robotology/icub-basic-demos) as per https://github.com/robotology/icub-main/issues/192.

Please take the corresponding actions.

**Comment by lornat75** [comment]
We should open an issue to enable compilation tests/dashboard otherwise nobody is checking compilation. In addition modules out of `icub-main` are not included with the binaries.
